src/types/WebLNProviderConfig.ts

```diff
@@ -0,0 +1,7 @@
+import {types} from '@getalby/sdk';
+
+export type WebLNProviderConfig = {
```

Comment:
can this be generic? 
These are options that a developer passes to an underlaying library. why does BC need to know about the potential options?

Reply:
I think it can to some extent, but we still need to know how to pass it - for example the NWC one if very specific right now (I don't know to do it generically yet, maybe we wait for a usecase?)